Taskfile.yml

```diff
@@ -258,23 +259,23 @@ tasks:
   controller:generate-crds:
     desc: Run controller-gen to generate {{.CONTROLLER_APP}} CRD files.
     deps: [controller:generate-types]
-    dir: "{{.CONTROLLER_ROOT}}"
     sources:
-    - "apis/**/*_gen.go" # depends on all generated types
-    - "pkg/genruntime/**/*.go" # Also depends on the genruntime types as they're embedded into the CRDs
+    - "v2/api/**/*_gen.go" # depends on all generated types
```

Comment:
Should we be using the `V2_ROOT` variable here?

Reply:
and below throughout

Reply:
I thought about it but is `{{.V2_ROOT}}` really better than `v2/`? I may even remove the variable

Reply:
It seems weird to use V2_ROOT in some places and not others though. I'm fine with either (agree the variable seems unneeded), but you use it below.